tests/e2e/vsphere_file_volume_basic_mount.go

```diff
@@ -142,7 +142,29 @@ var _ = ginkgo.Describe("[csi-file-vanilla] Verify Two Pods can read write files
 			1. Delete all the Pods, pvcs and storage class and verify the deletion
 	*/
 	ginkgo.It("[csi-file-vanilla] Verify Pod can read the files written by other Pod, which is deleted, when the Pod has pvc statically provisoned on same vsan file share", func() {
-		invokeTestForCreateFileVolumeAndMount(f, client, namespace, accessMode, filePath1, filePath2, true, false, true)
+		invokeTestForCreateFileVolumeAndMount(f, client, namespace, accessMode, filePath1, filePath2, true, false, true, true)
+	})
+	/*
+		Verify Pod can read the files written by other Pod, which is deleted, when both have same pvc mounted without specifing fstype in pv spec
+
+			1. Create StorageClass without specifying fsType
+			2. Create a PVC1 with "ReadWriteMany" using the SC from above
+		    3. Wait for PVC1 to be Bound
+			4. Get the VolumeID from PV
+		    5. Verify using CNS Query API if VolumeID retrieved from PV is present. Also verify
+				Name, Capacity, VolumeType, Health matches
+			6. Verify if VolumeID is created on one of the VSAN datastores from list of datacenters provided in vsphere.conf
+		    7. Create Pod1 using PVC1 created above at a mount path specified in PodSpec
+			8. Create a file (file1.txt) at the mount path. Check if the creation is successful
+			9. Delete the Pod and create a PVC2 with same file share created with PVC1
```

Comment:
Can you clarify this test step? I think here you are creating new static PV using file-share id from previously created Dynamic file-share PV?

Any particular reason we are creating new PV using the same file-share ID? Since PVC is ReadWriteMany, you can still create a new Pod with the same PVC on diff node.